department_app/rest/address_api.py

```diff
@@ -0,0 +1,153 @@
+"""Rest addresses Api"""
+from flask import Blueprint, request, jsonify, make_response
+from flask_restful import Resource, Api, abort
+from pydantic import ValidationError
+from department_app.models.app_models import Address
+from department_app.schemas.address_schema import AddressModel
+from department_app.models.app_models import db
+
+address_api = Blueprint('address_api', __name__)
+
+api = Api(address_api)
+
+
+class AddressInfo(Resource):
```

Comment:
For API endpoint names let's use the next naming:
- for the single entity - just a name of class (AddressInfo -> Address)
- for the list of entities - postfix 'list' (AllAddressesInfo -> AddressList)
And the same for all endpoints